README.md

```diff
@@ -32,6 +32,9 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+[TRELLO_API_KEY], [TRELLO_TOKEN] and [BOARD_ID] variables should be added to the `.env` file for accessing the Trello API
```

Comment:
It would be good to also mention here the various list ids

Reply:
List IDs were pulled via code as Dynamic variables. Have added card IDs, though, for the VCR tests